salt/states/file.py

```diff
@@ -3732,6 +3735,9 @@ def directory(name,
             file_mode = None
             dir_mode = None
 
+        if 'silent' in recurse_set:
+            ret['pchanges'] = 'Changes silenced'
```

Comment:
a while back we removed pchanges in favor of changes here https://github.com/saltstack/salt/pull/49832

can you change this to changes

Reply:
Done! It's a little confusing as master still has pchanges, but hopefully that's all that needs adjusting here.